assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/presentation/common/ApiResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.splab.invitation.presentation.common;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class ApiResponse<T> {
+
+  private Integer code;
+  private String message;
+  @JsonInclude(Include.NON_NULL)
+  private T result;
+
+  public static <T> ApiResponse<T> success() {
+    return new ApiResponse<>(ResponseType.OK.getCode(), ResponseType.OK.getMessage(), null);
```

Comment:
동일한 이름의 메서드의 다형성을 고려할 때는 인자가 적은 쪽에서 많은 쪽을 호출해가는 형식이 일반적입니다.
```suggestion
    return success(null)
```